### PR TITLE
Integration test for Bigtable init action.

### PR DESCRIPTION
**Pull request by szewi**
To run the test just use command:
  python -m unittest bigtable.test_bigtable

### CODE REVIEW

**Comment by szewi** [comment]
@karth295 @aniket486 could you take a look?
From all PRs with tests, I will suggest prioritizing this one. It changes the way we upload test scripts in integration_tests/dataproc_test_case.py file. Following PRs with tests will rely on this change.

File: bigtable/run_hbase_commands.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+import subprocess
+
+
+def create_commands_file():
+    file = open("commands", 'w+')
```
Comment (karth295) [inline bigtable/run_hbase_commands.py]: Nit: `with open(...) as file` (and then you don't need `file.close()`)

Comment (karth295) [inline bigtable/run_hbase_commands.py]: Nit: maybe `commands.txt` just to make it clear that it's a file. I was confused about `hbase shell commands` below

File: bigtable/test_bigtable.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+from parameterized import parameterized
+from integration_tests.dataproc_test_case import DataprocTestCase
+
+
+class BigTableTestCase(DataprocTestCase):
+    COMPONENT = 'bigtable'
+    INIT_ACTION = 'gs://dataproc-initialization-actions/bigtable/bigtable.sh'
+    TEST_SCRIPT_FILE_NAME = "run_hbase_commands.py"
+    METADATA = None
```
Comment (karth295) [inline bigtable/test_bigtable.py]: Nit: if these aren't constants they should be initialized in `__init__` as `self.lowercasevariable = None`

File: bigtable/test_bigtable.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        self.assertEqual(ret_code, 0, "Failed to validate cluster. Last error: {}".format(stderr))
+
+    def _validate_bigtable(self):
+        ret_code, stdout, stderr = self.run_command(
+            'cbt -instance {} read test-bigtable '.format(
+                self.DB_NAME
+            )
+        )
+        self.assertEqual(ret_code, 0, "Failed to validate cluster. Last error: {}".format(stderr))
+        self.assertIn("value1", stdout, "Failed to validate cluster. value1 is missing in the table")
```
Comment (karth295) [inline bigtable/test_bigtable.py]: Can you assert the exact value of the output?

File: bigtable/test_bigtable.py [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+            os.path.join(
+                os.path.dirname(os.path.abspath(__file__)),
+                self.TEST_SCRIPT_FILE_NAME
+            ),
+            name)
+        self._run_hbase_shell(name)
+        self._validate_bigtable()
+
+
+    @parameterized.expand([
```
Comment (karth295) [inline bigtable/test_bigtable.py]: Please add a comment on why 1.0 is not in the list

File: integration_tests/dataproc_test_case.py [hunk 5]
```diff
@@ -82,13 +82,9 @@ def tearDown(self):
...
 
-    def upload_test_file(self, name):
+    def upload_test_file(self, testfile, name):
         cmd = 'gcloud compute scp {} {}:'.format(
-            os.path.join(
-                os.path.dirname(os.path.abspath(__file__)),
-                self.COMPONENT.lower(),
-                self.TEST_SCRIPT_FILE_NAME
-            ),
+            testfile,
```
Comment (karth295) [inline integration_tests/dataproc_test_case.py]: If `TEST_SCRIPT_FILE_NAME` is no longer used in the base class, it should be removed from it

Comment (szewi) [reply integration_tests/dataproc_test_case.py]: ok, I will also modify other methods that are using this variable.

File: bigtable/test_bigtable.py (hunk 4 above)
Comment (szewi) [reply bigtable/test_bigtable.py]: 1.1 is also on the list but not working. I will remove 1.1 too adding proper comment.

File: bigtable/test_bigtable.py (hunk 3 above)
Comment (szewi) [reply bigtable/test_bigtable.py]: The output contains current timestamps which will be problematic to assert. Instead, I will use `count` function and compare its output with the expected number of rows.

File: bigtable/test_bigtable.py (hunk 2 above)
Comment (szewi) [reply bigtable/test_bigtable.py]: ok

File: bigtable/run_hbase_commands.py (hunk 1 above)
Comment (szewi) [reply bigtable/run_hbase_commands.py]: ok

Comment (szewi) [reply bigtable/run_hbase_commands.py]: using `with open` make a lot of sense when dealing with files. Thanks.

**Comment by szewi** [comment]
@karth295 this is ready for another round of review. Thanks.

**Pull request merged**